torchx/components/dist.py

```diff
@@ -261,6 +261,8 @@ def ddp(
         nnodes_rep,
         "--nproc_per_node",
         str(nproc_per_node),
+        "--node_rank",
+        f"{macros.replica_id}",
```

Comment:
can you add a comment that this only applies when rendezvous is static?

Reply:
Thanks @d4l3k. Done.